### DISCUSSION TITLE
Was a solution found for the MOTD problem? I'm unable to deploy because of it.

### DISCUSSION THREAD

**Discussion by RoudyTarabay** [deployphp/deployer#3640] [Help needed]
Hi everyone, 

As the title say is there any workaround or solutions for the MOTD related bug? I'm using hostinger and the option to remove the message doesn't seem to be available(or I didn't know how to do it)

Thanks!

**Comment by antonmedv** [comment]
Try to run this command for deploy user: 

    touch $HOME/.hushlogin

**Comment by metalinspired** [comment]
What worked for me was
```php
setShell('bash -norc -noprofile -ls')
```
